### PR TITLE
MQTT Vacuum docs

### PR DESCRIPTION
**Pull request by johnboiles**
**Basic MQTT vacuum support**

Useful for retrofitting older Roombas and other non-wifi vacuums to be controlled via Home Assistant. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9386

### CODE REVIEW

File: source/_components/vacuum.mqtt.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```yaml
+# Example minimal configuration.yaml entry
+vacuum:
+  - platform: mqtt
+```
+
+Configuration variables:
+
+- **name** (*Optional*): The name of the vacuum.
+- **supported_features** (*Optional*): List of features that the vacuum supports (possible values are `turn_on`, `turn_off`, `pause`, `stop`, `return_home`, `battery`, `status`, `locate`, `clean_spot`). Defaults to .
```
Comment (Landrash) [inline source/_components/vacuum.mqtt.markdown]: Default value is missing.

File: source/_components/vacuum.mqtt.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- **state_topic** (*Optional*): The MQTT topic subscribed to receive values from the vacuum. Defaults to `vacuum/state`.
+- **command_topic** (*Optional*): The MQTT topic to publish commands to control the vacuum. Defaults to `vacuum/command`.
+
+### {% linkable_title Retrofitting a non-wifi Roomba with an ESP8266 %}
+
+- [This repo](https://github.com/johnboiles/esp-roomba-mqtt) has MQTT client firmware for retrofitting your old Roomba
```
Comment (Landrash) [inline source/_components/vacuum.mqtt.markdown]: Dot is missing at the end of the sentence.

**Comment by frenck** [comment]
@Landrash Pff sorry, I've should not have missed that. I really need to get more sleep these days 😨

File: source/_components/vacuum.mqtt.markdown (hunk 1 above)
Comment (johnboiles) [reply source/_components/vacuum.mqtt.markdown]: Good catch!

File: source/_components/vacuum.mqtt.markdown (hunk 2 above)
Comment (johnboiles) [reply source/_components/vacuum.mqtt.markdown]: 👍

**Comment by johnboiles** [comment]
Hey @frenck @Landrash the corresponding PR oh home-assistant just got merged this morning. Could y'all take another look and merge if this looks good?

**Pull request merged**